integration_test/third_party_apps_test/applications/mysql/enable

```diff
@@ -19,7 +19,7 @@ metrics:
     mysql:
       type: mysql
       username: root
-      password: Ss123%321
+      password: {googlesecretmanager:projects/228437877067/secrets/ops-agent-3p-test-mysql-password/versions/1}
```

Comment:
Won't this cause the test to fail when run on any other project?

Also, why is there not a `$` at the start of the value?

Reply:
When does the test run on other projects? 

"Also, why is there not a $ at the start of the value?" -> fixed, ty.

Reply:
> When does the test run on other projects?

When it's run directly from a developer's workstation with `go test`

Reply:
The secrets are accessible as long as the SA attached to the test VM has `roles/secretmanager.secretAccessor` granted to the project hosting the secrets.

If the test is not running from within a VM, then the deverlopers `@google.com` email needs to be granted the iam role.

Reply:
And if the developer does not have an `@google.com` account? Ops Agent is open source software.